### PR TITLE
Set with as 100% when image embedded within two thirds container

### PR DESCRIPTION
**Pull request by thojava**
## Description
By default the image will have width as two third of its container. But when its parent is a `nhsuk-grid-column-two-thirds` then the image's width should be 100%.
## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] CHANGELOG entry

### CODE REVIEW

File: app/assets/sass/customize.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* Within the two-thirds containers the image should keep the full width */
```
Comment (smanivasag) [inline app/assets/sass/customize.scss:1]: Could we move this under `app/assets/sass/components/_images.scss` (so as to make/mark it as Images component specific change/customisation) and update it's import on `app/assets/sass/main.scss` ?

File: app/views/lks/copyright.html [hunk 2]
```diff
@@ -35,6 +35,13 @@ <h2>Copyright Act & 'Library Privilege'</h2>
             <p>If copies cannot be made under CLA Licence Plus, NHS library staff may supply copies under terms of the <a href="#">Copyright, Designs and Patents Act of 1988</a>. This includes digital as well as printed works.</p>
 
             <p>See also The British Library’s help about <a href="#">Library Privilege.</a></p>
+
+            <figure class="nhsuk-image">
```
Comment (smanivasag) [inline app/views/lks/copyright.html:39]: Could we replace the HTML with it's corresponding nunjucks macro (e.g. https://service-manual.nhs.uk/design-system/components/images) ?

```
{{ image({
  "src": "https://assets.nhs.uk/prod/images/S_1017_allergic-conjunctivitis_M15.2e16d0ba.fill-320x213.jpg",
  "alt": "Picture of allergic conjunctivitis",
  "caption": "Itchy, red, watering eyes"
}) }}
```

**Review by smanivasag — COMMENTED** [review]
Hi @thojava,

Just a thought - Found the following info from https://service-manual.nhs.uk/design-system/components/images under `How to use images` section:

> Images should flow with the text content, not appear too prominent or isolated.

Sorry, I haven't seen the figma/wireframe for Image component yet. But, making the Image to spread to the full width might make the image prominent (which apparently may not the original purpose of Image component based my understanding from the above info).

If the purpose of the Image component is to make it flow with the text around it, then we should probably use the Image component as is (without customising it to spread it to 100% width). Or if Images are isolated one and needs to be prominent on the page (with 100% width), then I would suggest to use the Card component (https://service-manual.nhs.uk/design-system/components/card#card-with-an-image) with Image in it. Let me know what you think. Thanks!

**Comment by thojava** [comment]
@smanivasag Here are the design https://www.figma.com/file/olZRiuJrZXfEaNeMMb7aNp/HEE-Designs?node-id=161%3A174 I don't think that should be a card. Do you think we need to discuss with the designer about this ?

**Comment by smanivasag** [comment]
Hi @thojava,

Thanks for sharing the figma design.

Fyi - With the following card component nunjucks snippet, I was able to get a similar look and feel indicated in the figma without any customisation:

```
{{ card({
  "imgURL": "https://assets.nhs.uk/prod/images/S_1017_allergic-conjunctivitis_M15.2e16d0ba.fill-320x213.jpg",
  "headingHtml": " ",
  "imgALT": "Picture of allergic conjunctivitis",
  "description": "Programmes, workouts and tips to get you moving and improve your fitness and wellbeing"
}) }}
```

But, semantically, if you think card doesn't qualifies to an Image component, then I am OK for you to use this PR to build Image component on brXM. And yes, it might be worth checking with the designer. Thanks!

**Comment by thojava** [comment]
@smanivasag I have refactored the sass structure a bit based on your comments. I think we still need this PR to make the frontend matched with the design.

**Review by smanivasag — APPROVED** [review]

**Pull request merged**